docs/xdr/features/collect/ingestion_methods/https.md

```diff
@@ -10,26 +19,29 @@ The following fields are currently handled by SEKOIA.IO’S HTTPS log collector:
 | `json`        | Yes        | String   | The actual log payload. If you want to push structured JSON logs, please send them as quoted JSON here |
 | `@timestamp`  | No         | Datetime | Event date if you want to push your own date (fallback is to use the reception’s date)                 |
 
-## Push Events to SEKOIA.IO Using Python
+
+When pushing your event, the collector returns the event's identifier in the SEKOIA.IO detection workflow, as a json payload.
 
 To push text events, one can just `POST` content to `https://intake.sekoia.io`:
 
 ```python
 import requests
 
 content = {"intake_key": "YOUR_INTAKE_KEY", "json": "[764008:0] info: 198.51.100.10 example.org. A IN"}
-requests.post("https://intake.sekoia.io", json=content)
+response = requests.post("https://intake.sekoia.io", json=content)
+print(response.text) // {"event_id": "uuid"}
```

Comment:
You can use code annotations here: https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-annotations

<pre>

```python
import requests
content = {"intake_key": "YOUR_INTAKE_KEY", "json": "[764008:0] info: 198.51.100.10 example.org. A IN"}
response = requests.post("https://intake.sekoia.io", json=content)
print(response.text) # (1)
```

1. Will print `{"event_id": "uuid"}`
```

Reply:
Changed